docs/autoupdate.md

```diff
@@ -0,0 +1,44 @@
+# Autoupdate
+
+*Autoupdate* is a feature that will guarantee you are always up to
+date. It is enabled by default, and can be disabled via `snappy config`.
+
+## Usage
+
+To check whether the feature is active, run
+
+    snappy config ubuntu-core | grep autoupdate
+
+If you want to disable it run
+
+    echo 'config: {ubuntu-core: {autoupdate: off}}' | sudo snappy config ubuntu-core
+
+and you then re-enable it via
+
+    echo 'config: {ubuntu-core: {autoupdate: on}}' | sudo snappy config ubuntu-core
+
+Every time autoupdate triggers it will try to update the whole system;
+if an `ubuntu-core` update is available the system will automatically
+reboot, although a message is printed to console with instructions on
+how to abort the reboot, in case you are logged in at the time.
+
+## Implementation details
+
+Autoupdate used to be called *autopilot* (but that got very confusing,
+especially when people were using snappy with other things that have
+their own autopilot, like an OpenStack deployment that used
+Canonical's own OpenStack Autopilot, or in mobile robots that could
+fly themselves); the `systemd` units still use this name.
```

Comment:
we could also rename the units, but I'm not at all sure it's worth it; rollback would be very tricky, and it's not really user-visible.